Makefile

```diff
@@ -70,25 +65,19 @@ gazelle: ## Run Bazel Gazelle
 check-install: ## Checks that you've installed this repository correctly
 	@./scripts/check-install.sh
 
-.PHONY: manager
-manager: generate  ## Build manager binary.
-	bazel build //cmd/manager $(BAZEL_ARGS)
-	install bazel-bin/cmd/manager/${BINARYPATHPATTERN}/manager $(shell go env GOPATH)/bin/aws-manager
-
 .PHONY: clusterctl
 clusterctl: generate ## Build clusterctl binary.
-	bazel build --workspace_status_command=./hack/print-workspace-status.sh //cmd/clusterctl $(BAZEL_ARGS)
-	install bazel-bin/cmd/clusterctl/${BINARYPATHPATTERN}/clusterctl $(shell go env GOPATH)/bin/clusterctl
+	bazel build //cmd/clusterctl $(BAZEL_ARGS)
 
 .PHONY: clusterawsadm
 clusterawsadm: dep-ensure ## Build clusterawsadm binary.
-	bazel build --workspace_status_command=./hack/print-workspace-status.sh //cmd/clusterawsadm $(BAZEL_ARGS)
-	install bazel-bin/cmd/clusterawsadm/${BINARYPATHPATTERN}/clusterawsadm $(shell go env GOPATH)/bin/clusterawsadm
+	bazel build //cmd/clusterawsadm/cmd/alpha/config:assets_bindata $(BAZEL_DOCKER_ARGS)
+	install bazel-genfiles/cmd/clusterawsadm/cmd/alpha/config/assets_vfsdata.go "cmd/clusterawsadm/cmd/alpha/config"
+	bazel build //cmd/clusterawsadm $(BAZEL_ARGS)
 
 .PHONY: release-binaries
 release-binaries: ## Build release binaries
-	bazel build --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 //cmd/clusterctl //cmd/clusterawsadm
-	bazel build --platforms=@io_bazel_rules_go//go/toolchain:darwin_amd64 //cmd/clusterctl //cmd/clusterawsadm
+	bazel build --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64,@io_bazel_rules_go//go/toolchain:darwin_amd64 //cmd/clusterctl //cmd/clusterawsadm
```

Comment:
Nice! is this new in bazel? Last time I tried to parse the docs it looked like --platforms would only take a singular value.